packages/edit-site/src/hooks/pattern-partial-syncing.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
We will need to support the connection setup in the standard post editor as well, eg. `/wp-admin/post.php?post=225&action=edit` for classic themes and users without siteadmin access.  So I wonder if we should move this to the patterns package and set the filter in for both instances?

Reply:
Good point! I completely forgot about that. I'll look into it.